spec/features/shelterreviews/shelter_review_index_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe "when I visit a shelter show page", type: :feature do
+  it "I see a list of reviews for that shelter" do
+
+    shelter1 = Shelter.create(name: "Dog-Haven",
```

Comment:
We might want to revisit some of these tests and get our _ put in or removed if that's how we want to go.  Polishing work though...